test/widgets/home_test.dart

```diff
@@ -215,6 +219,16 @@ void main () {
       check(find.byType(ProfilePage)).findsOne();
       check(find.text(eg.selfUser.fullName)).findsAny();
     });
+
+    testWidgets('_AboutZulipButton', (tester) async {
+      await prepare(tester);
+      await tapOpenMenu(tester);
+
+      await tester.tap(aboutZulipMenuIconFinder);
```

Comment:
nit:
```suggestion
      await tester.tap(find.byIcon(ZulipIcons.info));
```

The extra complication of being a descendant of BottomSheet is needed for three other items because they share their icons with corresponding buttons in the bottom nav, but isn't here.